test/integration/simple-deployment-annotated/01-assert.yaml

```diff
@@ -1,23 +1,8 @@
-apiVersion: lifecycle.keptn.sh/v1alpha3
```

Comment:
let's start counting from 00-* in the names as we always do

Reply:
this is an old test I just renamed the files, shall we do this anyhow?